### PR TITLE
feat(api): Allow only standard scripts in /push-tx

### PR DESCRIPTION
**Pull request by msbrogli**

### CODE REVIEW

File: hathor/transaction/resources/push_tx.py [hunk 1]
```diff
@@ -26,55 +26,70 @@ class PushTxResource(resource.Resource):
...
+        assert isinstance(tx, Transaction)
+        is_double_spending = tx.is_double_spending()
+        if is_double_spending:
+            return {
+                'success': False,
+                'message': 'Invalid transaction. At least one of your inputs has already been spent.',
+                'can_force': False
+            }
+
+        # Validate inputs.
```
[the remark this answers is not in the document]
Comment (msbrogli) [reply hathor/transaction/resources/push_tx.py:63]: Fixed. Thanks!

File: hathor/transaction/resources/push_tx.py [hunk 2]
```diff
@@ -26,55 +26,70 @@ class PushTxResource(resource.Resource):
     def __init__(self, manager):
         # Important to have the manager so we can know the tx_storage
         self.manager = manager
+        self.max_output_script_size: int = 256
```
[the remark this answers is not in the document]
Comment (msbrogli) [reply hathor/transaction/resources/push_tx.py]: Done. Thanks!

**Review by jansegre — APPROVED** [review]

**Review by pedroferreira1 — APPROVED** [review]

File: tests/resources/transaction/test_pushtx.py [hunk 3]
```diff
@@ -1,164 +1,188 @@
...
+
+            # Token creation tx
+            script_type_out = parse_address_script(blocks[0].outputs[0].script)
+            assert script_type_out is not None
+            address = script_type_out.address
+            tx2 = create_tokens(self.manager, address, mint_amount=100, propagate=False)
+            tx2_hex = tx2.get_struct().hex()
+            response = yield self.push_tx({'hex_tx': tx2_hex})
+            data = response.json_value()
+            print('!!', data)
```
Comment (pedroferreira1) [inline tests/resources/transaction/test_pushtx.py:124]: Debug print

**Review by pedroferreira1 — APPROVED** [review]

File: tests/resources/transaction/test_pushtx.py (hunk 3 above)
Comment (msbrogli) [reply tests/resources/transaction/test_pushtx.py]: Removed. Thanks!

**Review by jansegre — APPROVED** [review]

**Pull request merged**